deps/pcre.mk

```diff
@@ -23,7 +23,7 @@ $(BUILDDIR)/pcre2-$(PCRE_VER)/build-compiled: $(BUILDDIR)/pcre2-$(PCRE_VER)/buil
 	$(MAKE) -C $(dir $<) $(LIBTOOL_CCLD)
 	echo 1 > $@
 
-$(BUILDDIR)/pcre2-$(PCRE_VER)/checked: $(BUILDDIR)/pcre2-$(PCRE_VER)/build-compiled
+$(BUILDDIR)/pcre2-$(PCRE_VER)/build-checked: $(BUILDDIR)/pcre2-$(PCRE_VER)/build-compiled
```

Comment:
Is this change intentional?  It seems unrelated.

Reply:
It is. It was as missed update in 406af98ea16, but I guess nobody _really_ missed it.